src/coverlet.console/Program.cs

```diff
@@ -174,20 +176,55 @@ static int Main(string[] args)
                 }
 
                 var thresholdTypeFlags = ThresholdTypeFlags.None;
-
+                var thresholdTypeFlagQueue = new Queue<ThresholdTypeFlags>();
+                
                 foreach (var thresholdType in dThresholdTypes)
                 {
                     if (thresholdType.Equals("line", StringComparison.OrdinalIgnoreCase))
                     {
                         thresholdTypeFlags |= ThresholdTypeFlags.Line;
+                        thresholdTypeFlagQueue.Enqueue(ThresholdTypeFlags.Line);
                     }
                     else if (thresholdType.Equals("branch", StringComparison.OrdinalIgnoreCase))
                     {
                         thresholdTypeFlags |= ThresholdTypeFlags.Branch;
+                        thresholdTypeFlagQueue.Enqueue(ThresholdTypeFlags.Branch);
                     }
                     else if (thresholdType.Equals("method", StringComparison.OrdinalIgnoreCase))
                     {
                         thresholdTypeFlags |= ThresholdTypeFlags.Method;
+                        thresholdTypeFlagQueue.Enqueue(ThresholdTypeFlags.Method);
+                    }
+                }
+                
+                Dictionary<ThresholdTypeFlags, double> thresholdTypeFlagValues = new Dictionary<ThresholdTypeFlags, double>();
+                if (threshold.HasValue() && threshold.Value().Contains(','))
+                {
+                    var thresholdValues = threshold.Value().Split(',', StringSplitOptions.RemoveEmptyEntries).Select(t => t.Trim());
+                    if (thresholdValues.Count() != thresholdTypeFlagQueue.Count())
+                    {
+                        throw new Exception($"Threshold type flag count ({thresholdTypeFlagQueue.Count()}) and values count ({thresholdValues.Count()}) doesnt match");
```

Comment:
Could you end the exception with ```doesn't match.``` Also in ```CoverageResultTask```. Sorry for beeing petty :pray:.

Reply:
no problem, changed from `doesnt match` to `doesn't match` in both files. please check

Reply:
Thanks